src/rich-text-editor.component.js

```diff
@@ -28,7 +28,7 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
     // https://developer.mozilla.org/en-US/docs/Web/Events/paste
     event.preventDefault()
     event.stopPropagation()
-    let paste = richTextContext.sanitizeHTML((window.clipboardData || event.clipboardData).getData('text/plain'), 'pasteHTML')
+    let paste = richTextContext.sanitizeHTML((window.clipboardData || event.clipboardData).getData('text/html'), 'pasteHTML')
```

Comment:
`text/plain` makes it impossible to paste HTML from somewhere else, I was under the impression from the docs that this was supposed to be html.

See https://bandicoot.js.org/components/rich-text-editor.html#props and https://bandicoot.js.org/components/rich-text-editor.html#props

Both discuss the need to sanitize HTML when pasting.

Reply:
It is my understanding that this is safe to do.  Originally I was concerned setting it to 'text/html' would parse and potentially execute the html before sanitization occurred, but I believe that was a mistake. @joeldenning @dvnrsn Do either of you have reason to think otherwise?

Reply:
I think this is fine -- the html string is not executed until you actually append it to the dom